unified-codes/apps/web/src/components/containers/app/AppHeader.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '@material-ui/core/Link';
+
+import { IAlert } from '@unified-codes/data';
+import { AlertBar, AppBar, Avatar, Grid, UCIcon } from '@unified-codes/ui/components';
+import { withStyles, FlexDirection } from '@unified-codes/ui/styles';
+
+import { ITheme } from '../../../styles';
+
+import { AlertActions, IAlertAction } from '../../../actions';
+
+export interface HeaderProps {
+  classes?: {
+    root?: string,
+    title1?: string,
+    title2?: string,
+    menu?: string
+  },
+  alert: IAlert,
+  resetAlert: () => void,
+}
+
+const styles = (theme: ITheme) => ({
+  root: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.background.footer,
+    flexDirection: 'row' as FlexDirection,
+    padding: '12px 24px 12px 24px',
+    ' & div': {
+      flex: '0 1 0%',
+    },
+    boxShadow: 'none'
+  },
+  body: theme.typography.body1,
+  logo: {
+    fontWeight: 700,
+  },
+  menu: {
+    marginLeft: 150,
+    '& div ': {
+      paddingRight: 15,
+    },
+    '& a': {
+      color: 'rgba(255,255,255,0.83)',
+      fontSize: 14,
+      textTransform: 'uppercase',
+    },
+  },
+  title1: {
+    paddingRight: '5px!important',
+    ...theme.typography.subtitle1,
+  },
+  title2: {
+    paddingLeft: '5px!important',
+    ...theme.typography.subtitle2,
+  },
+});
+
+export type Header = React.FunctionComponent<HeaderProps>;
+
+const Header: Header = ({ classes, alert, resetAlert }) => {
```

Comment:
Still need to lift a few styles up here and add localisation, but not too much left here!